### DISCUSSION TITLE
show_pdf_page - check for empty PDF

### DISCUSSION THREAD

**Discussion by Andreasox** [pymupdf/PyMuPDF#3263] [Looking for help] [answered]
If trying to include an empty PDF the following error is raised:
  File "/Users/admin/PycharmProjects/pythonProject/.venv/lib/python3.9/site-packages/fitz/utils.py", line 166, in show_pdf_page
    raise ValueError("nothing to show - source page empty")
Is it possible with pyMuPDF to check if the source PDF is empty ?

**Comment by JorjMcKie** [comment, marked as answer]
This  message / exception is raised when the source page has no `/Contents` object. This can be checked with `page.get_contents() == []`.

**Reply by Andreasox** [reply]
Thanks. Is page.get_contents() documented?
 I cannot find it.

**Reply by JorjMcKie** [reply]
here https://pymupdf.readthedocs.io/en/latest/functions.html#Page.get_contents
